actions/postgresql_show_create_table_action.go

```diff
@@ -0,0 +1,448 @@
+	}
+	defer db.Close() //nolint:errcheck
+	var buf bytes.Buffer
+	w := bufio.NewWriter(&buf)
+
+	// Extract table id
+	tableID, err := e.printTableInit(ctx, w, db)
+	if err != nil {
+		return nil, err
+	}
+
+	/* Generate table cells to be printed */
+	err = e.printColumnsInfo(ctx, w, db, tableID)
+	if err != nil {
+		return nil, err
+	}
+
+	/* print indexes */
+	err = e.printIndexInfo(ctx, w, db, tableID)
+	if err != nil {
+		return nil, err
+	}
+
+	err = e.printCheckConstraints(ctx, w, db, tableID)
+	if err != nil {
+		return nil, err
+	}
+
+	err = e.printForeignKeyConstraints(ctx, w, db, tableID)
+	if err != nil {
+		return nil, err
+	}
+
+	err = e.printReferencedBy(ctx, w, db, tableID)
+	if err != nil {
+		return nil, err
+	}
+
+	if err = w.Flush(); err != nil {
+		return nil, err
+	}
+
+	return buf.Bytes(), nil
+}
+
+func (e *postgresqlShowCreateTableAction) printTableInit(ctx context.Context, w io.Writer, db *sql.DB) (string, error) {
+	var tableID, schema, relname string
+	row := db.QueryRowContext(ctx, `SELECT c.oid,
+	       n.nspname,
+	       c.relname
+	FROM pg_catalog.pg_class c
+	         LEFT JOIN pg_catalog.pg_namespace n ON n.oid = c.relnamespace
+	WHERE c.relname = $1
+	  AND pg_catalog.pg_table_is_visible(c.oid)
+	ORDER BY 2, 3;`, e.params.Table)
+	if err := row.Scan(&tableID, &schema, &relname); err != nil {
+		if err == sql.ErrNoRows {
+			return "", errors.Wrap(err, "Table not found")
+		}
+		return "", err
+	}
+	fmt.Fprintf(w, "Table \"%s.%s\"\n", schema, relname) //nolint:errcheck
+	return tableID, nil
+}
+
+func (e *postgresqlShowCreateTableAction) sealed() {}
+
+func (e *postgresqlShowCreateTableAction) printColumnsInfo(ctx context.Context, w io.Writer, db *sql.DB, tableID string) error {
+	rows, err := db.QueryContext(ctx, `SELECT a.attname,
+       pg_catalog.format_type(a.atttypid, a.atttypmod),
+       (SELECT substring(pg_catalog.pg_get_expr(d.adbin, d.adrelid) for 128)
+        FROM pg_catalog.pg_attrdef d
+        WHERE d.adrelid = a.attrelid
+          AND d.adnum = a.attnum
+          AND a.atthasdef),
+       a.attnotnull,
+       (SELECT c.collname
+        FROM pg_catalog.pg_collation c,
+             pg_catalog.pg_type t
+        WHERE c.oid = a.attcollation
+          AND t.oid = a.atttypid
+          AND a.attcollation <> t.typcollation)                          AS attcollation,
+       ''::pg_catalog.char                                               AS attidentity,
+       a.attstorage,
+       CASE WHEN a.attstattarget = -1 THEN NULL ELSE a.attstattarget END AS attstattarget,
+       pg_catalog.col_description(a.attrelid, a.attnum)
+FROM pg_catalog.pg_attribute a
+WHERE a.attrelid = $1
+  AND a.attnum > 0
+  AND NOT a.attisdropped
+ORDER BY a.attnum;`, tableID)
+	if err != nil {
+		return err
+	}
+
+	tw := tabwriter.NewWriter(w, 0, 0, 1, ' ', tabwriter.Debug)
+
+	fmt.Fprintln(tw, "Column\tType\tCollation\tNullable\tDefault\tStorage\tStats target\tDescription") //nolint:errcheck
+
+	for rows.Next() {
```

Comment:
We have to do two things:
* check `rows.Err()` after loop;
* `defer rows.Close()`.

Without them, the driver will leak resources that are critical on the client-side.
Do the same in other places.